lib/commands/install.ts

```diff
@@ -0,0 +1,72 @@
+///<reference path="../.d.ts"/>
+"use strict";
+
+import path = require("path");
+
+export class InstallCommand implements ICommand {
+	constructor(private $fs: IFileSystem,
+		private $errors: IErrors,
+		private $logger: ILogger,
+		private $options: IOptions,
+		private $injector: IInjector,
+		private $staticConfig: IStaticConfig) { }
+		
+	allowedParameters: ICommandParameter[] = [];
+	
+	execute(args: string[]): IFuture<void> {
+		return (() => {
+			let projectFilePath = this.getProjectFilePath(args[0]);
```

Comment:
Maybe you can cache the result of `this.getProjectFilePath(args[0])` in a variable, as you have already read the file and its content in `canExecute`.